Tests/NewsServiceTests/Validators/CurrencyRequestValidatorTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+
+using NUnit.Framework;
+using FluentValidation;
+
+using DTO.NewsRequests.Currency;
+using DTO.NewsRequests;
+using NewsService.Validators;
+using System.Linq;
+
+namespace NewsServiceTests.Validators
+{
+    public class CurrencyRequestValidatorTests
```

Comment:
what happens if you create an object CurrencyRequest without fields?

Reply:
look to 38-42 line